services/src/handlers/datasets.rs

```diff
@@ -37,7 +35,7 @@ use geoengine_operators::{
     engine::{StaticMetaData, VectorColumnInfo, VectorResultDescriptor},
     source::{
         OgrSourceColumnSpec, OgrSourceDataset, OgrSourceDatasetTimeType, OgrSourceDurationSpec,
-        OgrSourceTimeFormat,
+        OgrSourceErrorSpec, OgrSourceTimeFormat,
```

Comment:
the handlers shouldn't use any types from the operators crate. At least not as part of the API

Reply:
These types are not part of the API. Before they reach the API they get converted to types from api::model. This is done at the end of `suggest_meta_data_handler`.